types/react-reconciler/test/ReactTestHostConfig.ts

```diff
@@ -1,189 +1,197 @@
-import * as TestRendererScheduling from './ReactTestRendererScheduling';
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```

Comment:
It seems like it would be more thorough to copy the license file too, but there does happen to be an MIT license in the root of this repo.